SimGeneral/MixingModule/python/digitizers_cfi.py

```diff
@@ -104,10 +106,9 @@
 
 from Configuration.ProcessModifiers.run3_ecalclustering_cff import run3_ecalclustering
 (run3_ecalclustering | phase2_hgcal).toModify( theDigitizersValid,
-                       calotruth = cms.PSet( caloParticles ) ) # Doesn't HGCal need these also without validation?
+                       calotruth = cms.PSet( caloParticles ) )
 (premix_stage2 & phase2_hgcal).toModify(theDigitizersValid, calotruth = dict(premixStage1 = True))
 
-
 phase2_timing.toModify( theDigitizersValid.mergedtruth,
```

Comment:
should check with @cms-sw/mtd-dpg-l2 if this behavior is needed for non-pdigi_valid workflow